build.gradle

```diff
@@ -71,7 +71,7 @@ allprojects {
   apply plugin: 'nebula.ospackage-base'
 
   group = 'io.snappydata'
-  version = '1.0.2-SNAPSHOT'
+  version = '1.0.2'
```

Comment:
We will have an RC build first and then within 10 days should have the 1.0.2

Reply:
Ok